js/page.js

```diff
@@ -169,9 +169,9 @@
       let inputSelector = ".newtab-control, .newtab-link";
       for (let input of document.querySelectorAll(inputSelector)) {
         if (aValue) {
-          input.removeAttribute("tabindex");
+          delete input.data.tabindex;
         } else {
-          input.setAttribute("tabindex", "-1");
+          input.data.tabindex = -1;
```

Comment:
This is a real attribute (controls the browser's tabbing behavior), so shouldn't be "data". So this one we should leave as is.